src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -326,12 +327,12 @@ DataView::ActionStatus LiveFunctionsDataView::GetActionStatus(
 
 void LiveFunctionsDataView::OnIteratorRequested(const std::vector<int>& selection) {
   for (int i : selection) {
-    uint64_t instrumented_function_id = GetInstrumentedFunctionId(i);
+    uint64_t scope_id = GetScopeId(i);
     const FunctionInfo* instrumented_function = GetFunctionInfoFromRow(i);
-    const ScopeStats& stats =
-        app_->GetCaptureData().GetScopeStatsOrDefault(instrumented_function_id);
+    ORBIT_CHECK(instrumented_function != nullptr);
```

Comment:
Why this cannot be a `nullptr`?

Reply:
Currently, it can, but only in devmode. The check is added for the sake of a log message. I plan to make the action of adding an iterator inactive (and even invisible) for anything what is not a hooked function, so this will never be null.

Reply:
>  I plan to make the action of adding an iterator inactive (and even invisible) for anything what is not a hooked function

Do you have a bug for that? If so, you can add a TODO here.